client/html/layouts/basket/mini/product-body-default.html

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @copyright Copyright (c) Metaways Infosystems GmbH, 2013
+ * @license LGPLv3, http://opensource.org/licenses/LGPL-3.0
+ */
+
+
+try
+{
+	$miniPriceValue = $this->mainPriceItem->getValue();
+    $miniPriceCosts = $this->mainPriceItem->getCosts();
+	$miniPriceCurrency = $this->translate( 'client/html/currency', $this->mainPriceItem->getCurrencyId() );
+}
+catch( Exception $e )
+{
+	$miniPriceValue = '0.00';
+	$miniPriceCurrency = '';
+}
+
+
+
+$products = $this->get( 'mainProducts', 0 );
+
+
+/// Price format with price value (%1$s) and currency (%2$s)
+$priceFormat = $this->translate( 'client/html', '%1$s %2$s' );
+
+$enc = $this->encoder();
+?>
+<div class="basket">
+	<ul>
+		<div class="basket-mini-main">
+			<?php $i=0;?>
+			<?php foreach($products as $product) { ?>
```

Comment:
Please use the alternative PHP syntax only and move the "<?php" to the front if they are the first strings on the line:
<?php foreach($products as $product) : ?>
...
<?php endforeach; ?>